src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-notification-list/rgw-bucket-notification-list.component.scss

```diff
@@ -0,0 +1,5 @@
+@use'@carbon/layout';
+
+::ng-deep.cds--type-mono{
+ margin-right: layout.$spacing-02;
+}
```

Comment:
if you directly modify the copy2-clipboard component's scss file then we might not need to add the `:ng-deep`?